docs/runbooks/duplicate_roles_in_course.md

```diff
@@ -17,7 +17,7 @@ python manage.py lms shell --settings=aws
 >>> dup
 [<Role: Administrator for MITx/6.041r_3/2016_Spring>, <Role: Moderator for MITx/6.041r_3/2016_Spring>, <Role: Community TA for MITx/6.041r_3/2016_Spring>, <Role: Student for MITx/6.041r_3/2016_Spring>]
 >>> for r in dup:
-... r.delete()
+...     r.delete()
 ...
 >>>
 ```
```

Comment:
Is this file supposed to be here?

Reply:
That was because I had changed the file and accidentally committed it. I can take it out.